relay-event-normalization/src/normalize/mod.rs

```diff
@@ -847,9 +870,10 @@ pub struct LightNormalizationConfig<'a> {
 
     /// Configuration for measurement normalization in transaction events.
     ///
-    /// If provided, normalization truncates custom measurements and adds units of known built-in
-    /// measurements.
-    pub measurements_config: Option<&'a MeasurementsConfig>,
+    /// Has an optional [`MeasurementsConfig`] from both the project and the global level.
+    /// If at least one is provided, then normalization will truncate custom measurements
+    /// and add units of known built-in measurements.
+    pub dynamic_measurements_config: Option<DynamicMeasurementsConfig<'a>>,
```

Comment:
Could we call this field just `measurements`? Both the `dynamic` and `config` name components do not help to increase clarity.